packages/components/src/components/navigation-item/navigation-item.ts

```diff
@@ -0,0 +1,297 @@
+              this.calculatePaddingX()
+            )}
+          ></slot>`
+        : null;
+
+    const childrenSlot = slots['children'] && !this.horizontal ? html`<slot name="children"></slot>` : null;
+
+    /* eslint-disable lit/no-invalid-html */
+    /* eslint-disable lit/binding-positions */
+    const root = html`
+      ${hostDisplayStyle}
+
+      <${tag}
+        part="base"
+        class=${cx(
+          'hover:bg-neutral-200 transition-all min-h-[48px] cursor-pointer relative',
+          this.horizontal ? 'border-b-4 px-4' : 'border-l-4 pl-[28px] pr-8',
+          this.current ? 'border-accent' : 'border-transparent',
+          this.disabled ? 'text-neutral-500 border-neutral-500 pointer-events-none' : 'text-primary',
+          { base: 'text-base', larger: 'text-lg', smaller: 'text-[14px]' }[this.size],
+          this.isAccordion() ? 'flex flex-col' : 'inline-block w-full'
+        )}
+        aria-controls=${ifDefined(this.isAccordion() ? 'navigation-item-details' : undefined)}
+        aria-current=${ifDefined(this.current ? 'page' : undefined)}
+        aria-disabled=${this.disabled}
+        ?disabled=${ifDefined(this.isButton() ? this.disabled : undefined)}
+        href=${ifDefined(this.href || undefined)}
+        role=${this.isLink() ? 'link' : 'button'}
+        tabindex=${this.disabled ? '-1' : '0'}
+        @click=${this.isAccordion() ? this.handleClickSummary : this.isButton() ? this.handleClickButton : undefined}
+        @mouseenter=${ifDefined(this.disabled ? undefined : this.handleMouseEnter)}
+        @mouseleave=${ifDefined(this.disabled ? undefined : this.handleMouseLeave)}
+      >
+        <span class=${cx(
+          'relative pt-3 inline-flex justify-between items-center',
+          this.isAccordion() ? 'grow' : 'w-full',
+          slots['description'] ? 'pb-1' : horizontalPaddingBottom,
+          this.calculatePaddingX()
+        )}>
+          <span class="inline-flex items-center flex-auto">
+            ${iconLeftSlot}
+            <slot part="label" class=${cx('inline', slots['icon-right'] || slots['main'] ? 'mr-2' : '')}></slot>
+            ${iconRightSlot}
+          </span>
+          <span class="inline-flex items-center">
+            ${mainSlot}
+            ${chevron}
+          </span>
+        </span>
+        ${descriptionSlot}
+      </${tag}>
+    `;
+    /* eslint-enable lit/no-invalid-html */
+    /* eslint-enable lit/binding-positions */
+
+    return this.isAccordion()
+      ? html`${divider}
+          <details id="navigation-item-details" ?open=${this.open} class="relative flex">
+            ${root}${childrenSlot}
+          </details>`
+      : html`${divider}${root}`;
+  }
+
+  /**
+   * Inherits Tailwindclasses and includes additional styling.
+   */
+  static styles = [
+    componentStyles,
+    SolidElement.styles,
+
+    css`
+      :host {
+        box-sizing: border-box;
+        position: relative;
+      }
+
+      :host > * {
+        text-align: left;
+      }
```

Comment:
Should not be part of the component but come from the style above. As this miht have led to problems in the stories, you just have to add some CSS for the stories, but not in the component.